### PR TITLE
Fix bugs and changed file name

### PR DESCRIPTION
**Pull request by razvanazamfirei**
## Description

Updated API call to new endpoint, changed add method to "quick add" to reduce the number of arguments, cleaned up redundant code

## Type of change

- [x] Bug fix
- [X] Improvement of an existing script

## Checklist

- [X] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/todoist/create-task.sh [hunk 1]
```diff
@@ -1,41 +1,36 @@
...
-# @raycast.needsConfirmation false
-# @raycast.argument1 { "type": "text", "placeholder": "Buy Milk", "optional": false, }
-# @raycast.argument2 { "type": "text", "placeholder": "Due (Defaults to Tomorrow)", "optional": true, }
-
 
 # Optional parameters:
 # @raycast.icon images/todoist-logo.png
+# @raycast.packageName Todoist
+# @raycast.needsConfirmation false
+# @raycast.argument1 { "type": "text", "placeholder": "call mom tomorrow at 5", "optional": false, }
```
Comment (unnamedd) [inline commands/apps/todoist/create-task.sh:12]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Call Mom Tomorrow at 5" }
```

File: commands/apps/todoist/create-task.sh [hunk 2]
```diff
@@ -1,41 +1,36 @@
...
+
+# Documentation: 
+# @raycast.description Creates Todoist task
```
Comment (unnamedd) [inline commands/apps/todoist/create-task.sh:15]:
```suggestion
# @raycast.description Create Task
```

File: commands/apps/todoist/create-task.sh [hunk 3]
```diff
@@ -1,41 +1,36 @@
 #!/bin/bash
```
Comment (unnamedd) [inline commands/apps/todoist/create-task.sh:1]: This Script Command demands an extra setup, for that reason, you must rollback the file name to `create-task.template.sh`, the suffix `.template` for this case is mandatory in this repository.

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @razvanazamfirei,

thanks for your update in this Script Command. I've made some comments, please, let us know in the comment when you address them for a second round of review.

**Comment by razvanazamfirei** [comment]
Should be addressed. Thank you!

**Pull request merged**